### PR TITLE
Adding LanguageExtractor Class

### PR DESCRIPTION
**Pull request by anshul0708**
Adding LanguageExtractor Class, searching from middle of the web pages, using 500 chars.

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -39,18 +40,25 @@ Downloader.downloadFile(crawlBaseUrl + fileName, dataFolder, (err, filepath) =>
             console.log("unpacking complete!");
         }
 
-        // TODO: Language filter
+        // Extract english pages
+        LanguageExtractor.extractWETPages("./data/CC-MAIN-20170116095119-00016-ip-10-171-10-70.ec2.internal.warc.wet", 'eng', (err,filepath) => {
```
Comment (lgfrbcsgo) [inline src/app.ts]: Remove the file path. `"./data/CC-MAIN-20170116095119-00016-ip-10-171-10-70.ec2.internal.warc.wet"`,
`filepath` already contains the path for the downloaded and unpacked file. Just replace it with `filepath` ;)

Comment (anshul0708) [inline src/app.ts]: Opps! Left over from some testing.

Comment (lgfrbcsgo) [inline src/app.ts]: No problem 😄

File: src/language-extractor.ts [hunk 2]
```diff
@@ -0,0 +1,68 @@
+/**
+ * Created by Anshul on 3/7/2017.
+ */
```
Comment (kordianbruck) [inline src/language-extractor.ts]: These comments aren't really need?

Comment (anshul0708) [inline src/language-extractor.ts]: Yeah, these were added by Intellij automatically when creating the file. I'll remove these.

**Comment by sacdallago** [comment]
And delete the branch also please
||
||

**Pull request merged**